packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -15,6 +15,7 @@
   box-sizing: border-box;
   display: flex;
   position: relative;
+  border: 1px solid transparent;
```

Comment:
Are we concerned with the height of the message bar changing with that extra border? Is there a reason for not using outline?